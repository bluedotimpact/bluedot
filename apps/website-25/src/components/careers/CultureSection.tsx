import { Section } from '@bluedot/ui';

const CultureSection = () => {
  return (
    <Section className="culture-section" title="Our culture">
      <div className="culture-section__container flex lg:flex-row flex-col gap-space-between">
        <div className="culture-section__text-container flex flex-col gap-5">
          <p>At BlueDot Impact, our culture is defined by high agency, rapid experimentation, and direct feedback. We move fast and iterate quickly because AI capabilities are advancing rapidly and our work is time-sensitive.</p>
          <p>But speed doesn't mean sacrificing support â€“ we invest heavily in each other's growth and wellbeing, creating an environment where ambitious, mission-driven people can do their best work.</p>
          <p>We seek exceptional people who combine a sense of urgency with the empathy to help others thrive, whether that's fellow team members or the thousands of professionals in our global community.</p>
        </div>
        <img className="culture-section__image max-w-[570px] w-full h-auto rounded-2xl" src="/images/team/team_1.jpg" alt="BlueDot Impact team" />
      </div>
    </Section>
  );
};

export default CultureSection;
