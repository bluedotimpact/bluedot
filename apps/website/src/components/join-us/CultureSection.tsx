import { Section, SectionHeading } from '@bluedot/ui';
import { P } from '../Text';

const CultureSection = () => {
  return (
    <Section className="culture-section">
      <div className="culture-section__container flex lg:flex-row flex-col gap-space-between">
        <div className="culture-section__text-container flex flex-col gap-5 max-w-[436px]">
          <SectionHeading title="Our culture" />
          <P>At BlueDot Impact, our culture is defined by high agency, rapid experimentation, and direct feedback. We move fast and iterate quickly because AI capabilities are advancing rapidly and our work is time-sensitive.</P>
          <P>But speed doesn't mean sacrificing support â€“ we invest heavily in each other's growth and wellbeing, creating an environment where ambitious, mission-driven people can do their best work.</P>
          <P>We seek exceptional people who combine a sense of urgency with the empathy to help others thrive, whether that's fellow team members or the thousands of professionals in our global community.</P>
        </div>
        <div className="culture-section__image-grid grid mt-6 grid-cols-2 gap-x-2 gap-y-4 sm:mt-0 sm:grid-cols-3 sm:gap-4">
          <img className="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm" src="/images/culture/culture_dewi_v1.jpeg" alt="BlueDot Impact team" />
          <img className="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm" src="/images/culture/culture_talking_v2.jpeg" alt="BlueDot Impact team" />
          <img className="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm" src="/images/culture/culture_cap_v1.jpeg" alt="BlueDot Impact team" />
          <img className="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm" src="/images/culture/culture_will_v1.jpeg" alt="BlueDot Impact team" />
          <img className="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm" src="/images/culture/culture_josh_v1.jpg" alt="BlueDot Impact team" />
          <img className="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm" src="/images/culture/culture_keyboard_v1.jpeg" alt="BlueDot Impact team" />
        </div>
      </div>
    </Section>
  );
};

export default CultureSection;
