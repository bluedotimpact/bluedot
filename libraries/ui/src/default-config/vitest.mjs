/* eslint-disable import/no-extraneous-dependencies */
import { defineConfig, defaultExclude } from 'vitest/config';
import react from '@vitejs/plugin-react';
import * as dotenv from 'dotenv';

/**
 * @param {import('vitest/config').UserConfig} [config] 
 * @returns {import('vitest/config').UserConfig}
 */
export const withDefaultBlueDotVitestConfig = async (config) => defineConfig({
  ...config,
  plugins: [react(), ...(config?.plugins ?? [])],
  test: {
    environment: 'happy-dom',
    env: dotenv.config({ path: '.env.test' }).parsed,

    ...config?.test,

    exclude: [...defaultExclude, '**/.{turbo,next}/**', ...(config?.test?.exclude ?? [])],
  },
});
