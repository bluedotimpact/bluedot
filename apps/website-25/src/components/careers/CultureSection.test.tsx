import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';
import CultureSection from './CultureSection';

describe('CultureSection', () => {
  test('renders default as expected', () => {
    const { container } = render(<CultureSection />);
    expect(container).toMatchSnapshot();
  });
});
