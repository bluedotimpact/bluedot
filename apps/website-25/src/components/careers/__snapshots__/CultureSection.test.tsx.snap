// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CultureSection > renders default as expected 1`] = `
<div>
  <section
    class="section section-body culture-section"
  >
    <div
      class="section__body"
    >
      <div
        class="culture-section__container flex lg:flex-row flex-col gap-space-between"
      >
        <div
          class="culture-section__text-container flex flex-col gap-5 max-w-[436px]"
        >
          <div
            class="section-heading__title-container flex justify-between items-center gap-space-between mb-6"
          >
            <div
              class="section-heading__content flex-1 flex flex-col gap-2"
            >
              <h2
                class="section-heading__title relative"
              >
                Our culture
              </h2>
            </div>
          </div>
          <p>
            At BlueDot Impact, our culture is defined by high agency, rapid experimentation, and direct feedback. We move fast and iterate quickly because AI capabilities are advancing rapidly and our work is time-sensitive.
          </p>
          <p>
            But speed doesn't mean sacrificing support â€“ we invest heavily in each other's growth and wellbeing, creating an environment where ambitious, mission-driven people can do their best work.
          </p>
          <p>
            We seek exceptional people who combine a sense of urgency with the empathy to help others thrive, whether that's fellow team members or the thousands of professionals in our global community.
          </p>
        </div>
        <div
          class="culture-section__image-grid grid mt-8 grid-cols-2 gap-x-2 gap-y-4 sm:mt-0 sm:grid-cols-3 sm:gap-4"
        >
          <img
            alt="BlueDot Impact team"
            class="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm"
            src="/images/culture/culture_dewi_v1.jpeg"
          />
          <img
            alt="BlueDot Impact team"
            class="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm"
            src="/images/culture/culture_talking_v1.jpeg"
          />
          <img
            alt="BlueDot Impact team"
            class="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm"
            src="/images/culture/culture_happy_v1.jpeg"
          />
          <img
            alt="BlueDot Impact team"
            class="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm"
            src="/images/culture/culture_will_v1.jpeg"
          />
          <img
            alt="BlueDot Impact team"
            class="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm"
            src="/images/culture/culture_prod-eng_v1.jpeg"
          />
          <img
            alt="BlueDot Impact team"
            class="culture-section__image max-h-[100px] sm:max-h-[178px] w-full object-cover rounded-sm"
            src="/images/culture/culture_keyboard_v1.jpeg"
          />
        </div>
      </div>
    </div>
  </section>
</div>
`;
